crates/stackable-operator/src/commons/s3/crd.rs

```diff
@@ -64,14 +64,8 @@ pub struct S3ConnectionSpec {
     /// NOTE: This is not the bucket region, and is used by the AWS SDK to
     /// construct endpoints for various AWS service APIs. It is only useful when
     /// using AWS S3 buckets.
-    ///
-    /// When using AWS S3 buckets, you can configure optimal AWS service API
-    /// connections in the following ways:
-    /// - From **inside** AWS: Use an auto-discovery source (eg: AWS IMDS).
-    /// - From **outside** AWS, or when IMDS is disabled, explicity set the
-    ///   region name nearest to where the client application is running from.
     #[serde(default)]
-    pub region: AwsRegion,
+    pub region: Region,
```

Comment:
Note: comments need updating since region means everything (both client and bucket region) to AWS :/

Reply:
In the context of S3, this will primarily mean the bucket region (the client region uses the same setting, but the bucket regions MUST be set correctly for valid sigv4 headers).